README.md

```diff
@@ -376,18 +376,27 @@ This repository contains a curated list of awesome open source libraries that wi
 * [CuML](https://github.com/rapidsai/cuml) ![](https://img.shields.io/github/stars/rapidsai/cuml.svg?style=social) - cuML is a suite of libraries that implement machine learning algorithms and mathematical primitives functions that share compatible APIs with other RAPIDS projects.
 * [CuPy](https://github.com/cupy/cupy) ![](https://img.shields.io/github/stars/cupy/cupy.svg?style=social) - An implementation of NumPy-compatible multi-dimensional array on CUDA. CuPy consists of the core multi-dimensional array class, cupy.ndarray, and many functions on it.
 * [DeepSpeed MII](https://github.com/microsoft/DeepSpeed-MII) ![](https://img.shields.io/github/stars/microsoft/DeepSpeed-MII.svg?style=social) - DeepSpeed MII makes low-latency and high-throughput inference possible, powered by DeepSpeed.
+* [fastai](https://github.com/fastai/fastai) ![](https://img.shields.io/github/stars/fastai/fastai.svg?style=social) - fastai is a deep learning library which provides practitioners with high-level components that can quickly and easily provide state-of-the-art results in standard deep learning domains, and provides researchers with low-level components that can be mixed and matched to build new approaches.
 * [Flax](https://github.com/google/flax) ![](https://img.shields.io/github/stars/google/flax.svg?style=social) - A neural network library and ecosystem for JAX designed for flexibility.
+* [fastai](https://github.com/fastai/fastai) ![](https://img.shields.io/github/stars/fastai/fastai.svg?style=social) - fastai is a deep learning library which provides practitioners with high-level components that can quickly and easily provide state-of-the-art results in standard deep learning domains, and provides researchers with low-level components that can be mixed and matched to build new approaches.
+* [GPyTorch](https://github.com/cornellius-gp/gpytorch) ![](https://img.shields.io/github/stars/cornellius-gp/gpytorch.svg?style=social) - GPyTorch is a Gaussian process library implemented using PyTorch. GPyTorch is designed for creating scalable, flexible, and modular Gaussian process models with ease.
```

Comment:
Doesn't seem computation optimization more than specialisation to scientific domain

Reply:
Thanks @axsaucedo I recently participated in the OSS commit to Jina AI and found they are using GPytorch to accelerate the deep learning relevant deployment, as also specified in their documentation:
![image](https://user-images.githubusercontent.com/8592144/222985814-5e21b8ad-554e-49fb-8ddb-088122bf5d17.png)

I think your question is actually if Gaussian learning is a subcategory of machine learning and I indeed had the same concern when I decide to add it to our list. I found they are together as indicated below:
![image](https://user-images.githubusercontent.com/8592144/222985949-77c216c8-4a41-4b0b-9c85-f4e64305d43c.png)

Reply:
Quesiton is not whether Gaussian learning is a subcategory of machine learning, it's on whether framework is just an implementation of Pytorch for a specialised domain as opposed for computation optimisation, it seems it does become quite gray area, but let's merge for now